java-chassis-reference/en_US/general-development/local-develop-test.md

```diff
@@ -24,65 +24,68 @@
-   1. 下载服务注册中心可执行文件压缩包并解压缩
+    (1) Download the [Service Registry Executable Compressor] (http://apache.org/dyn/closer.cgi/incubator/servicecomb/incubator-servicecomb-service-center/1.0.0-m1/apache-servicecomb- Incubating-service-center-1.0.0-m1-windows-amd64.tar.gz)  
+    (2) Extract to the current folder  
+    (3) Go to the unzipped directory and double-click to run the **start-service-center.bat** file.  
+  
+      </div>
+      <div id="linux" class="tab-pane fade" markdown="1">
+    1) Download the Service Registry executable file archive and extract it  
    ```bash
    wget http://apache.org/dyn/closer.cgi/incubator/servicecomb/incubator-servicecomb-service-center/1.0.0-m1/apache-servicecomb-incubating-service-center-1.0.0-m1-linux-amd64.tar.gz
    tar xvf apache-servicecomb-incubating-service-center-1.0.0-m1-linux-amd64.tar.gz
-   ```
-   2. 运行服务注册中心
-   ```bash
-   bash apache-servicecomb-incubating-service-center-1.0.0-m1-linux-amd64/start-service-center.sh
-   ```
-   
-    注意：前端（frontend）在Linux环境下默认会绑定ipv6地址，导致浏览器报错，修复办法为：先修改conf/app.conf中的httpaddr为外部可达网卡ip，之后修改app/appList/apiList.js中`ip : 'http://127.0.0.1'`为对应ip，最后重启ServiceCenter即可。
-  
-    </div>
-   </div>
+  ```  
+   2) Run the service registry  
+   ```bash
+   Bash apache-servicecomb-incubating-service-center-1.0.0-m1-linux-amd64/start-service-center.sh
+   ```  
+   
+    Note: The frontend (frontend) will be bound to the ipv6 address by default in the Linux environment, causing the browser to report an error. The repair method is: first modify the httpaddr in conf/app.conf to the external reachable network card ip, and then modify the app/appList/apiList. .js `ip : 'http://127.0.0.1'` for the corresponding ip, and finally restart ServiceCenter.
+  
+    </div>
+   </div>
 
-   注意：Window和Linux版本均只支持64位系统。
+   Note: Both Windows and Linux versions only support 64-bit systems.  
 
-2. 以Docker的方式运行
+2. Run as Docker  
 
 ```bash
-docker pull servicecomb/service-center
-docker run -d -p 30100:30100 servicecomb/service-center:latest
+Docker pull servicecomb/service-center
+Docker run -d -p 30100:30100 servicecomb/service-center:latest
 ```
 
-* **步骤 2 **启动本地服务中心后，在服务提供/消费者的microservice.yaml文件中配置ServerCenter的地址和端口，示例代码：
+* **Step 2 ** After starting the local service center, configure the ServerCenter address and port in the service provider/consumer's microservice.yaml file. Example code:
 
 ```yaml
-servicecomb:
-  service:
-    registry:
-      address:
-        # 服务中心地址及端口
-        http://127.0.0.1:30100
+Servicecomb:
+  Service:
+    Registry:
+      Address:
+        #Service Center address and port
+        Http://127.0.0.1:30100
 ```
 
-* **步骤 3 **开发服务提供/消费者，启动微服务进行本地测试。
+* **Step 3 **Development service provider/consumer, launch microservices for local testing.
 
-**----结束**
+**----End**
 
-## Mock机制启动服务中心
-在本进程内存中模拟一个只能本进程使用的服务中心，一般是在测试场景中使用。
-* ### 进程内调用
-只需要在启动ServiceComb引擎之前声明一下即可启用：
+## Mock mechanism start service center
```

Comment:
Mock mechanism start service center ->
Using mock mechanism to start ServiceCenter